projects/npm-tools/packages/npm-scripts/src/utils/getExternalExportsWebpackConfigs.js

```diff
@@ -79,6 +79,27 @@ module.exports = function getExternalExportsWebpackConfigs(
 				},
 			};
 
+			if (process.env.NODE_ENV === 'development') {
```

Comment:
I've moved this to the top so that I can disable source maps for SVG files, since they don't serve any purpose.